README.md

```diff
@@ -108,6 +108,19 @@ To install this module from source, run the following commands:
  - make test
  - make install
 
+### From checkout
+
+This module uses Dist::Zilla, you can install from a checkout using the following commands:
```

Comment:
You forgot to add "you might want to grab a drink because this will take 8 hours."